js/modules/k6/ws/ws.go

```diff
@@ -454,22 +454,19 @@ func (s *Socket) closeConnection(code int) error {
 
 // Wraps conn.ReadMessage in a channel
 func readPump(conn *websocket.Conn, readChan chan []byte, errorChan chan error, closeChan chan int) {
-	defer func() { _ = conn.Close() }()
```

Comment:
This `Close()` seemed redundant since it's already done in `socket.Close()`, and this `*Read*Message` wrapper shouldn't be closing connections anyway IMO.